use_case_examples/cifar_brevitas_training/evaluate_torch_cml.py

```diff
@@ -8,34 +8,25 @@
 from torch.utils.data import DataLoader
 from torchvision import transforms
 from torchvision.datasets import CIFAR10
+from tqdm import tqdm
 from trainer import accuracy
 
 from concrete.ml.torch.compile import compile_brevitas_qat_model
 
 
-def cml_inference(quantized_numpy_module, x_numpy):
-    predictions = np.zeros(shape=(x_numpy.shape[0], 10))
-    for idx, x in enumerate(x_numpy):
-        x_q = np.expand_dims(x, 0)
-        predictions[idx] = quantized_numpy_module.forward(x_q, fhe="simulate")
-    return predictions
-
-
 def evaluate(torch_model, cml_model):
-    # Import the CIFAR data (following bnn_pynq_train.py)
 
+    # Transform pipeline that normalizes the data between -1 and 1
     transform_to_tensor = transforms.Compose(
         [
             transforms.ToTensor(),
             transforms.Lambda(lambda x: 2.0 * x - 1.0),
-        ]  # Normalizes data between -1 and +1s
+        ]
     )
 
-    builder = CIFAR10
-
-    test_set = builder(root=".datasets/", train=False, download=True, transform=transform_to_tensor)
-
-    test_loader = DataLoader(test_set, batch_size=128, shuffle=False, num_workers=1)
+    # Import and load the CIFAR test dataset (following bnn_pynq_train.py)
+    test_set = CIFAR10(root=".datasets/", train=False, download=True, transform=transform_to_tensor)
+    test_loader = DataLoader(test_set, batch_size=128, shuffle=False, num_workers=0)
```

Comment:
I put "num_workers=0" because 1 made the script failed on my side : 
```
AttributeError: Can't pickle local object 'evaluate.<locals>.<lambda>'
```